tests/decimals.json

```diff
@@ -1,9 +1,9 @@
-{"n":"\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
-{"n":"\u00FF\u00FF\u00FF\u00FF\u00FF\u00FF\u00FF\u00FF\u00FF\u00FF\u00F4\u00D0.\u00DE\u0088\u0000"}
-{"n":"\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u000B/\u00D1!x\u0000"}
-{"n":"\u0000\u0000\u0000\u0000\u0003\u00FD5\u00EBk\u00C2\u00DFF\u0018\b\u0000\u0000"}
-{"n":"\u00FF\u00FF\u00FF\u00FF\u00FF\u00FF\u00FF\u0093u\u00E4\u00AC\u001C8\u00E4\u0000\u0000"}
-{"n":"\u00FF\u00FF\u00FF\u00FF\u00FF\u00FF\u00FDb\u00C6\u00B5\u00A2\u009C\u00FA\u00BC\u0000\u0000"}
-{"n":"\u00FF\u00FF\u00FF\u00FF\u00FF\u00FF\u00F1\u0012/\u00F5\u00A5\u00020\u00F0\u0000\u0000"}
-{"n":"\u00FF\u00FF\u00FF\u00FF\u00FF\u00FF\u00FE\u0081\u00D1\u0098\u0090\u00808\u0018\u0000\u0000"}
-{"n":"\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0096\u0093h\u0097L\u0005\u00B0\u0000\u0000"}
+{"n":"[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]"}
```

Comment:
Shouldn't this be an array (and not a string)?

Reply:
Fixed now